### PR TITLE
Add tests for 3 and 3.2 branches, support 5.6 and test master

### PR DESCRIPTION
**Pull request by tractorcow**
Since master doesn't support 5.3, this is added separately as a test vs php 5.6

### CODE REVIEW

**Comment by tractorcow** [comment]
Hm, master isn't any good, since composer restricts it. I've removed this for now.

Need to look into the 3.x / 3.2 errors.

**Comment by tractorcow** [comment]
It looks like these broke because `ManyMany` is added as a new physical method in 3.2, so you can't really have relations named after that method anymore. ;)

**Comment by tractorcow** [comment]
@nyeholt hi marcus, do you think you'd have a moment to look at this? This fix is necessary for 3.2 compatibility.

The code itself is fine, but the tests won't pass without these fixes. ;P

File: .travis.yml [hunk 1]
```diff
@@ -1,12 +1,20 @@
 language: php
 
-php:
-  - 5.3
+sudo: false
+
+php: 
+ - 5.3
```
Comment (assertchris) [inline .travis.yml]: 2 spaces to 1? SHAME ON YOU!

**Comment by tractorcow** [comment]
Fixed @assertchris

**Comment by nyeholt** [comment]
Thanks! Am a bit behind the 8 ball (and 9, and 10) at the moment :)

**Comment by tractorcow** [comment]
Yay thanks!

**Pull request merged**